Makefile

```diff
@@ -325,3 +324,19 @@ install-gateway-api-local:
 .PHONY: uninstall-gateway-api
 uninstall-gateway-api:
 	kubectl delete -f $(GATEWAY_API_CRDS_LOCAL_PATH)
+
+### dev-env:			  Launch development environment
+.PHONY: dev-env
+dev-env: kind-up pack-image
+	helm repo add apisix https://charts.apiseven.com
+	helm repo update
+	helm install apisix apisix/apisix \
+		--set gateway.type=NodePort \
+		--set ingress-controller.enabled=true \
+		--namespace ingress-apisix \
+		--create-namespace
+	kubectl set image deployment/apisix-ingress-controller ingress-controller=$(REGISTRY)/apache/apisix-ingress-controller:dev --namespace ingress-apisix
+
```

Comment:
Hi, can u try this?

```makefile
### dev-env:			  Launch development environment
.PHONY: dev-env
dev-env: kind-up pack-image
	-helm repo add apisix https://charts.apiseven.com 2>&1 | grep -q "Error: repository name (apisix) already exists" && echo "Apisix already exists, skipping installation" || \
		(helm repo update && \
		 helm list -n ingress-apisix | grep -q apisix && echo "Apisix already exists, skipping installation" || \
		 helm install apisix apisix/apisix \
			--set gateway.type=NodePort \
			--set ingress-controller.enabled=true \
			--namespace ingress-apisix \
			--create-namespace && \
		 kubectl set image deployment/apisix-ingress-controller ingress-controller=$(REGISTRY)/apache/apisix-ingress-controller:dev --namespace ingress-apisix)

```

Reply:
I guess newer versions of helm automatically detect if the repository exists and skips over without throwing any error. 
Also if errors are to be handled, maybe it would be cleaner to be a script file just like `kind-up`?

Reply:
Of course, it is my habit to make mistakes in this place. 

Perhaps we need @tao12345666333 's pieces of advice.

Reply:
Perhaps we don't need to verify the release, just upgrade directly. 
```shell
helm upgrade --install apisix apisix/apisix \
		--set gateway.type=NodePort \
		--set ingress-controller.enabled=true \
		--namespace ingress-apisix \
		--create-namespace
```

Reply:
Thanks for all your suggestions, I will try is later.

Reply:
@Chever-John @tao12345666333 I update the code to skip installation if apisix already exists, and to keep Makefile clean, I moved the installation code to separated shell script `utils/install-apisix-with-helm.sh`, please review. Thanks.

Reply:
https://github.com/apache/apisix-ingress-controller/pull/1460/commits/bd8a97be1163f2dbceabdfd94abfa1894f43a95f